syncopy/shared/tools.py

```diff
@@ -48,6 +48,42 @@ def __str__(self):
             ppStr = "{}"
         return ppStr
 
+    def copy(self):
+        """
+        Create a shallow-copy of this StructDict instance.
+
+        Note: Overwrites the `.copy` method of the parent `dict` class, otherwise `copy()` will return a `dict` instead of a `StructDict`.
+
+        Returns
+        -------
+        Shallow-copy of StructDict.
+
+        Notes
+        -----
+        Consider calling the `.deepcopy()` method to get a deep copy instead, which is typically what users want.
+        """
+        obj = type(self).__new__(self.__class__)
+        obj.__dict__.update(self.__dict__)
+        return obj
+
+    def deepcopy(self):
```

Comment:
Wondering if we should just wire `copy()` to this deepcopy method as a default?! I think from a user perspective shallow copies are hardly a useful feature, even though shallow copies are more or less Python's default?